subsys/bluetooth/audio/ascs.c

```diff
@@ -47,24 +48,77 @@
 struct bt_ascs_ase {
 	struct bt_ascs *ascs;
 	struct bt_audio_ep ep;
+	const struct bt_gatt_attr *attr;
 	struct k_work work;
+	atomic_t ref;
 };
 
 struct bt_ascs {
 	struct bt_conn *conn;
-	struct bt_ascs_ase ases[ASE_COUNT];
+	struct bt_ascs_ase *ases[ASE_COUNT];
```

Comment:
Did you consider to use `sys_slist` instead of fixed-size array of pointers?

Reply:
Unresolving this as I have the same question :) Did you get an answer?

Reply:
Sorry, I missed this comment initially. I changed the implementation to use `sys_slist` instead